NAM/get_dsp.cpp

```diff
@@ -161,25 +162,31 @@ std::unique_ptr<DSP> get_dsp(dspData& conf)
     for (size_t i = 0; i < config["dilations"].size(); i++)
       dilations.push_back(config["dilations"][i]);
     const std::string activation = config["activation"];
-    return std::make_unique<convnet::ConvNet>(
-      loudness, channels, dilations, batchnorm, activation, params, expected_sample_rate);
+    return haveLoudness ? std::make_unique<convnet::ConvNet>(
```

Comment:
The ternary expressions are more words but clearer here and we're not giving a fake loudness to the object, which I think is a net positive